runtimes/rust/Cargo.toml

```diff
@@ -25,11 +25,13 @@ exonum-proto = { version = "0.13.0-rc.2", path = "../../components/proto" }
 failure = "0.1.5"
 futures = "0.1.25"
 log = "0.4.8"
-protobuf = "2.8.0"
+protobuf = "2.10.1"
 serde = "1.0.101"
 serde_derive = "1.0.101"
 
 [dev-dependencies]
+exonum-testkit = { version = "0.13.0-rc.2", path = "../../test-suite/testkit" }
```

Comment:
`exonum-testkit` depends on `exonum-rust-time`, and `exonum-rust-runtime` has `exonum-testkit` as a dev dependency.

Won't it cause problems with uploading to `crates.io`?

Reply:
Good point. I ran into a similar case some time ago and I used this [workaround](https://github.com/rust-lang/cargo/issues/4242#issuecomment-413203081).

Reply:
I see.

Reply:
Wouldn't a better solution be creating a separate non-publishable crate for testing (like `consensus-tests` and `node-tests` before)? As far as I see, the tested functionality is public, so it is technically possible.

Reply:
Hopefully it would.